minor styling changes

katieefrey opened the pull request:
I added a line break above the "cut/paste" links so there is a little more white space.

Also I removed the line specifying the color for links within the widget and the underline text decoration in ejp_uat.css, that way the widget can match the overall styling of the site hosting the widget (aka I think the blue links look bad on my website and I decided that I am not a fan of the double underline effect that was for some reason created by the underline text-decoration).

katieefrey committed:
added a line break above the cut/paste links for more white space, removed coloring for widget links in css

katieefrey committed:
removed extra underline as well...

pkgw commented:
This looks good, but I kept in the special styling for the links in the widget because they're a bit funky: they don't have an `href` attribute, so by default (in Firefox at least) they don't get styled like links. This means that unless the person using the widget adds some special CSS, the default appearance is going to be a bit funky.

Ooh, I have an idea for a workaround — I'll explore and push it to this PR if it looks promising.

pkgw committed:
src/ejp_uat.js: add href="javascript:void(0);" to generate <a> tags

Without an `href` attribute, browsers won't style them as active links. This
can be worked around in CSS, but it means that users of the widget are going
to need to implement some custom CSS, which we'd like to avoid. Based on the
relevant StackOverflow discussion
(https://stackoverflow.com/questions/4855168/what-is-href-and-why-is-it-used),
it looks like the best solution is to use this sort of `javascript:` href. If
the widget user wants to customize the styling of these links with CSS, they
still can, of course, but this should yield nicer behavior by default.

pkgw commented:
OK, I think the workaround works — see the commit message with 9fbc631.

katieefrey commented:
Whatever workaround you did looks great from my prescriptive!  Thanks!
